### ISSUE TITLE
Transcoding files with special characters broken in AWS

### ISSUE THREAD

**Issue by phuongdh** [avalonmediasystem/avalon#4287]
@SolomonHD reported this error

```
	in 41 seconds	2	default	ActiveEncode::PollingJob	
{"_aj_globalid"=>"gid://ActiveEncode/ElasticTranscoderEncode/1600371363476-0wwjmz"}
URI::InvalidURIError: bad URI(is not URI?): s3://stack1-test-derivatives/39f1aa88-bd5a-4e7b-b319-246036db1995/quality-low/[MP4_720p]_D16_ROCKER_ARM_VALVETRAIN_ASSAMBLY_honda_civic.mp4
```

### Done Looks Like
- Renaming for files before transcoding includes removal/replacement of bracket characters

**Comment by joncameron** [comment]
https://docs.aws.amazon.com/AmazonS3/latest/userguide/object-keys.html

**Comment by joncameron** [comment]
Merged to #3926
